### DISCUSSION TITLE
Diff comparing files too

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1823] [Suggestions]
Hello everyone.
As in title, please, add the feature to compare text files (other than folders) with builtin comparison tool.
Something you'd do with WinMerge, but with DoubleCMD, which I like more.

Thanks in advance for any help.
Regards, have a nice day.

**Comment by JonnyTech** [comment]
https://doublecmd.github.io/doc/en/help.html#cm_CompareContents

**Comment by anonimo82** [comment]
Ok, very sorry
